src/researchhub_document/related_models/constants/document_type.py

```diff
@@ -28,6 +28,9 @@
 FILTER_HAS_BOUNTY = "FILTER_HAS_BOUNTY"
 FILTER_OPEN_ACCESS = "FILTER_OPEN_ACCESS"
 FILTER_PEER_REVIEWED = "FILTER_PEER_REVIEWED"
+FILTER_HUBS_EXCLUDED = "FILTER_HUBS_EXCLUDED"
+FILTER_EXCLUDED_FROM_FEED = ("FILTER_EXCLUDED_FROM_FEED",)
+FILTER_INCLUDED_IN_FEED = ("FILTER_INCLUDED_IN_FEED",)
```

Comment:
```suggestion
FILTER_EXCLUDED_FROM_FEED = "FILTER_EXCLUDED_FROM_FEED"
FILTER_INCLUDED_IN_FEED = "FILTER_INCLUDED_IN_FEED"
```

Reply:
Right, you know what, I'm not sure why I did it this way. I almost swear linting formatted things like this lol